packages/teams-js/webpack.config.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
       type: 'umd',
       umdNamedDefine: true,
     },
+    globalObject: 'this',
```

Comment:
Why was this necessary? I believe you that it is (because we talked about it) but it would be goo to capture in a comment in the code or on the PR so that future developers will know what it is for

Reply:
Originally this was fixing an issue where webpack was compiling our .ts files to `self` which was conflicting with some node.js stuff. I did some more testing, and it seems since I now have nextjs working properly, it's no longer necessary, so I removed it. If I ever get new errors regarding `self` being undefined, I'll re-add it.